save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/Warning.kt

```diff
@@ -57,6 +57,32 @@ internal fun String.extractWarning(warningRegex: Regex,
     )
 }
 
+/**
+ * @param warningRegex regular expression for warning
+ * @param fileName file name
+ * @param line line number of warning
+ * @param message waring text
+ * @param columnGroupIdx index of capture group for column number
+ * @return a [Warning] or null if [this] string doesn't match [warningRegex]
+ */
+@Suppress("TOO_MANY_PARAMETERS", "LongParameterList")
+internal fun String.extractWarning(warningRegex: Regex,
```

Comment:
May be it's better to add some flag, instead of creation a new function? the logic almost the same

Reply:
Yes, the methods have the same logic, but they have different signatures, so I think method overloading is better here.